packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -254,22 +250,24 @@ export class DbSyncStakePoolProvider extends DbSyncProvider(RunnableModule) impl
       this.#builder.queryPoolHashes(query, params)
     );
     // Get cached total staked amount
-    const totalStake = await this.#cache.get(queryCacheKey(StakePoolsSubQuery.TOTAL_STAKE), async () => {
-      const distribution = await this.cardanoNode.stakeDistribution();
+    const totalStake = this.#useBlockfrost
+      ? '1'
```

Comment:
Why do we use '1' here? Some defaulting if Blockfrost is used?

Reply:
That value is unused when blockfrost is used, any value is fine.

Reply:
Maybe const `STAKE_NOT_USED = '0' | 'NotUsed'` ?

Reply:
9d34b4639 should solve this

Reply:
`null` typically represents absence of value. Why not just use it? `queryExtraPoolsData` argument type can also be updated to `string | null` - then it's more explicit.

Reply:
@IvayloAndonov , @mkazlauskas 's suggestion is even better.
abb9b0178 should solve this